### ISSUE TITLE
<a11y>The ARIA role 'separator' is not valid for the element <input>

### ISSUE THREAD

**Issue by xubinnb** [carbon-design-system/ibm-products#4032]
===== ENVIRONMENT =====
Browser: Version 105.0.5195.125 (Official Build) (arm64)
Automatically update Chrome for all users Learn more
Test Tool Type: IBM Accessbility Checker

===== DEFECT DESCRIPTION ===== 
Problem 1: The ARIA role 'separator' is not valid for the element
Problem 2: An element with ARIA role 'separator' does not have the required ARIA attribute(s): 'aria-valuenow'

=====STEPS TO REPRODUCE =====
Go to https://dev.console.test.cloud.ibm.com/projects

Use IBM Equal Access Accessibility Checker to scan

===== EXPECTED RESULT =====
[IBM 4.1.2 Name, Role, Value](https://www.ibm.com/able/requirements/requirements/#4_1_2)
For all user interface components (including but not limited to: form elements, links, and components generated by scripts), the name and role can be programmatically determined; states, properties, and values that can be set by the user can be programmatically set; and notification of changes to these items is available to user agents, including assistive technologies.

===== SCREEN CAPTURE =====
<img width="1268" alt="image" src="https://media.github.ibm.com/user/42629/files/8fe62374-3e2d-4455-833c-e36809d3c46c">
<img width="1276" alt="image" src="https://media.github.ibm.com/user/42629/files/d4ac4c6f-dae8-41e6-8030-316e7ce20ca2">
<img width="1268" alt="image" src="https://media.github.ibm.com/user/42629/files/eb0369c5-a97d-4bbc-9572-d27a42ef9d64">
<img width="1270" alt="image" src="https://media.github.ibm.com/user/42629/files/5078d216-db7c-4655-ab1c-0047deb14db6">
```[tasklist]
### Acceptance criteria
- [ ] Investigate alternative element for `input`
```

**Comment by davidmenendez** [comment]
hey @xubinnb this should have been fixed in a previous commit https://github.com/carbon-design-system/ibm-products/pull/4072

please update to the latest version of carbon for ibm products and let us know if this problem still persists! thanks.

**Comment by elycheea** [comment]
@SeonyuK did some investigation and we may not have resolved this issue. Even though we removed `role="separator"` in our implementation, it’s still visible in Storybook and in latest versions of our package. The separator role seems to be getting set via `react-table`[^1].

Suggest we explore options from my previous comment for this one rather than just deleting the role.

[^1]:https://github.com/TanStack/table/blob/v7/src/plugin-hooks/useResizeColumns.js

**Comment by davidmenendez** [comment]
issue has been resolved. please keep an eye out for our latest release and let us know if anything else comes up.

**Comment by SeonyuK** [comment]
> issue has been resolved. please keep an eye out for our latest release and let us know if anything else comes up.


@davidmenendez 
We updated it to `2.44.0-rc.1` but still can see the issue.
The weird thing is I can't see the issue in your storybook with the same version. 
Is there something we need to set up? 
In your storybook the separator role.

This is your storybook
![Screenshot 2024-07-03 at 10 33 02 AM](https://github.com/carbon-design-system/ibm-products/assets/144151347/f3383931-d401-4820-a9f7-bde839bee607)
 is in the <span>  not <input> but ours it is still in the <input>


This is our 
![Screenshot 2024-07-03 at 10 34 32 AM](https://github.com/carbon-design-system/ibm-products/assets/144151347/72c4e394-f37a-45a9-bf5d-5be45cd732a4)
